packages/tldraw/src/lib/canvas/TldrawOverlays.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEditor, useEditorComponents, useValue } from '@tldraw/editor'
+import { getArrowTargetState } from '../shapes/arrow/arrowTargetState'
+import { DraggingHandle } from '../tools/SelectTool/childStates/DraggingHandle'
+import { PointingHandle } from '../tools/SelectTool/childStates/PointingHandle'
+
+/** @public @react */
+export function TldrawOverlays() {
```

Comment:
it's worth adding a comment here and why the arrow hints are on their own layer as opposed to another existing layer. i think we might forget in future refactoring